src/components/deploymentConfig/DeploymentConfig.utils.ts

```diff
@@ -185,3 +186,47 @@ export const getPosition = (isValues: boolean, isEnv: boolean, type: number) =>
         if (type === 2) return 2
     }
 }
+
```

Comment:
Please ignore these functions for a while it will be discussed today whether this logic will be there in frontend or will be handled in backend